extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -700,6 +705,50 @@ export class SubscriptionService {
     return null;
   }
 
+  /**
+   * Start deferred account creation flow if service config has
+   * appropriate configuration.
+   * @return {?Promise}
+   * @private
+   */
+  maybeStartDeferredAccountFlow_() {
+    return this.platformStore_
+      .getEntitlementPromiseFor('local')
+      .then((localEntitlement) => {
+        if (localEntitlement.grantReason == GrantReason.SUBSCRIBER) {
```

Comment:
Should we, conversely, also require that the granting entitlement is `GrantReason.SUBSCRIBER`? Otherwise we don't know if that even that service has an account.

Reply:
Done.